identity-service/latest/tutorial/sso/ldap.md

```diff
@@ -108,43 +108,13 @@ The properties listed that need to be set for Alfresco Content Services (ACS) ar
     |csrf.filter.referer | The referer value of ACS to prevent Cross Site Request Forgery (CSRF), for example `https://repo.example.com`|
     |csrf.filter.origin | The origin value of ACS to prevent Cross Site Request Forgery (CSRF), for example `https://repo.example.com/*`|
 
-2. Update the `share-config-custom.xml` file located by default in `$ALFRESCO_HOME/tomcat/shared/classes/alfresco/web-extension/`:
+2. Sign in to the administrator console of ACS as an administrator. The URL of the administrator console is `https://repo.example.com:443/alfresco/service/enterprise/admin`.
 
-    * Set the `CSRFPolicy` to true as in the following example:
```

Comment:
We need to remove only the second bullet point from this step - `* Add the following two rules to allow for signing out via SAML:`.

Reply:
Fixed